web3.js/src/transaction/legacy.ts

```diff
@@ -195,7 +195,7 @@ export class Transaction {
   /**
    * The first (payer) Transaction signature
    */
-  get signature(): Buffer | null {
+  get signature(): Uint8Array | null {
```

Comment:
This is unfortunately a breaking change, because `Buffer` is an instance of a `Uint8Array`, but not the other way around. This change would strip functionality away from the return type by making it less specific.